lib/rules/no-render-return-value.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Prevent usage of the return value of React.render
+ * @author Dustan Kasten
+ */
+'use strict';
+
+var versionUtil = require('../util/version');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+module.exports = function(context) {
+
+  // --------------------------------------------------------------------------
+  // Public
+  // --------------------------------------------------------------------------
+
+  return {
+
+    CallExpression: function(node) {
+      var callee = node.callee;
+      var parent = node.parent;
+      if (callee.type !== 'MemberExpression') {
+        return;
+      }
+
+      var calleeObjectName;
+      if (versionUtil.test(context, '15.0.0')) {
```

Comment:
how would you recommend doing that? `versionUtil` seems to be `true` for any greater version. I initially had:

```
var calleeObjectName = /^ReactDOM$/;
if (versionUtil.test(context, '0.13.0')) {
  // ...
} else if (versionUtil.test(context, '0.14.0')) {
  // ...
}
```